CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.0.0-alpha.5
+
+* `FEAT`: validate cloud element templates against JSON Schema ([#561](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/561))
+* `FEAT`: use components instead of elements for all entries ([#590](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/590))
+* `FIX`: set correct title to element templates select ([#591](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/591))
+* `DEPS`: update to `@bpmn-io/properties-panel@0.11.0`
+* `DEPS`: update to `@bpmn-io/element-templates-validator@0.5.0`
+
+### BREAKING CHANGES
+
+* `component` property of an entry must be an actual component, not an element

```

Comment:
Let's link some guidance here. I think the original PR could be sufficient.

Reply:
👍  I added it accordingly.